uiowa/uiowa
# Feature: Fix for update block button visibilty on windows

# How to test

- `yarn workspace uids_base build`
- `blt ds --site=sandbox.uiowa.edu`
- Test on browserstack Chrome and verify that you can see the "update" block if editing a block in layout builder
